ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Keystore/Util.hs

```diff
@@ -0,0 +1,97 @@
+-- | system file operations related to key-store UserSecret
+
+module Ariadne.Wallet.Cardano.Kernel.Keystore.Util
```

Comment:
This module is related to I/O operations on the new `UserSecret`. Perhaps a better name for this module would be something like `Persistence` or `IO`?